### PR TITLE
W-14581754-recorder 6.5.2 service release rns

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+|RPA Builder | 6.4.0 and later
+|===
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|When recording a screenshot using the *Full screen* option in the *Define screenshot* pop-up, RPA Recorder no longer crashes when you resume recording or add an activity. |W-14573347
```
Comment (Alexander-S-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]: The root of the problem here was the the Recording Toolbar was displayed during recording screenshots **manually**. The possibility to resume recording or to add an activity would not be available if the toolbar would not be displayed.
Therefore I suggest to combine both issues as something like "The Recording Toolbar is not displayed during adding manual screenshots anymore, until adding screenshot is finished".

**Review by Alexander-S-Salesforce — DISMISSED** [review]

**Review by Alexander-S-Salesforce — DISMISSED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+= RPA Recorder 6.5.2 Release Notes
+
+RPA Recorder enables you to record a business process for automation.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== December 5, 2023
+
+// == Compatibility
+include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+2]
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+[%header%autowidth.spread]
+|===
+|Application |Version
+|RPA Builder | 6.4.0 and later
+|===
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|The Recording Toolbar no longer displays while adding manual screenshots. Once screenshots are confirmed, the toolbar displays again. |W-14573347, W-14572817
+|RPA Recorder no longer creates subfolder `RPARecorder` in the `RPARecorder` folder when upgrading from version 6.4.4 to 6.5.2. |W-14600072
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
|RPA Recorder no longer creates the `RPARecorder` subfolder in the `RPARecorder` folder when upgrading from version 6.4.4 to 6.5.2. |W-14600072
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+|===
+
+// == See Also
+include::partial$release-notes/rn-headings.adoc[tag=seeAlso,leveloffset=+1]
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
include::partial$release-notes/rn-headings.adoc[tag=seeAlso,leveloffset=+2]
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+* https://help.mulesoft.com/s/[MuleSoft Help Center]
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
* https://help.mulesoft.com/s/[MuleSoft Help Center^]
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+* https://www.mulesoft.com/support-and-services[MuleSoft Services]
```
Comment (sathya0) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc]:
```suggestion
* https://www.mulesoft.com/support-and-services[MuleSoft Services^]
```

**Review by sathya0 — APPROVED** [review]

**Pull request merged**